server-api/src/main/java/org/apache/atlas/RequestContext.java

```diff
@@ -86,6 +85,7 @@ public class RequestContext {
     private String      currentTypePatchAction = "";
     private AtlasTask   currentTask;
     private String traceId;
+    private final Map<AtlasObjectId, Object> relationshipEndToVertexIdMap = new HashMap<>();
```

Comment:
relationshipEndToVertexIdMap - what does relationshipEnd mean?

Reply:
It is the relationshipEnd AtlasObjectId.